src/content/en/2020/mobile-web.md

```diff
@@ -5,20 +5,295 @@ chapter_number: 12
 title: Mobile Web
 description: Mobile Web chapter of the 2020 Web Almanac covering page loading, textual content, zooming and scaling, buttons and links, and ease of filling out forms.
 authors: [spanicker, mdiblasio]
-reviewers: [malchata, obto, cheneytsai]
+reviewers: [obto]
 analysts: [obto]
 translators: []
-#spanicker_bio: TODO
-#mdiblasio_bio: TODO
+spanicker_bio: Shubhie Panicker is the engineering lead for Chrome’s engagement in the web framework ecosystem, where she collaborates with open source tools, frameworks and communities. As a member of Chrome’s Web Platform team she has worked on web standards and chromium’s implementation for several web performance APIs. Prior to Chrome, she worked on infrastructure and web frameworks for Google products like Search, Google Photos etc. <a href="https://twitter.com/shubhie">Twitter handle</a>
```

Comment:
This last bit sticks out, I'd suggest removing it as you'll have a Twitter icon/link under your name anyway. (per your [contributor config](https://github.com/HTTPArchive/almanac.httparchive.org/blob/d46be2f17b334ded616827b332e554c8f108482e/src/config/2020.json#L1224))
```suggestion
spanicker_bio: Shubhie Panicker is the engineering lead for Chrome’s engagement in the web framework ecosystem, where she collaborates with open source tools, frameworks and communities. As a member of Chrome’s Web Platform team she has worked on web standards and chromium’s implementation for several web performance APIs. Prior to Chrome, she worked on infrastructure and web frameworks for Google products like Search, Google Photos etc.
```